swapserverrpc/server.proto

```diff
@@ -715,4 +724,90 @@ message ServerWithdrawResponse {
 
     // The nonces that the server used to generate the sweepless sweep sigs.
     repeated bytes server_nonces = 2;
+}
+
+message ServerStaticAddressLoopInRequest {
+    // The client's key for the htlc output.
+    bytes htlc_client_key = 1;
+
+    // The hashed swap invoice preimage of the swap.
+    bytes swap_hash = 2;
+
+    // The deposit outpoints the client wishes to loop in. They implicitly state
+    // the swap amount.
+    repeated string deposit_outpoints = 3;
+
+    // The swap invoice that the client wants the server to pay.
+    string swap_invoice = 4;
+
+    // An optional last hop the client wants to receive the invoice payment
+    // from.
+    bytes last_hop = 5;
+
+    // The protocol version that the client adheres to.
+    StaticAddressProtocolVersion protocol_version = 6;
+
+    // The user agent string that identifies the software running on the user's
+    // side. This can be changed in the user's client software but it _SHOULD_
+    // conform to the following pattern:
+    //    Agent-Name/semver-version(/additional-info)
+    // Examples:
+    //    loopd/v0.10.0-beta/commit=3b635821
+    //    litd/v0.2.0-alpha/commit=326d754
+    string user_agent = 7;
```

Comment:
Again thanks for poking, I was using the initator as the user agent, but there's actually a function that appends the right info so that it matches the examples. Have a look at  https://github.com/lightninglabs/loop/blob/master/version.go#L57. Will update the PR

Reply:
Reminder to update the field description.

Reply:
added an example `loopd/v0.10.0-beta/commit=3b635823,initiator=easy-autoloop`